.changes/next-release/Bug Fix-a8470482-60c4-4711-b967-e5b8c446a027.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Fix a bug when pressing up or down does not move cursor when suggestion is active"
```

Comment:
- The "Bug Fix" type will already list the item as a fix. No need to repeat "fix a bug" in the description. Just describe the bug or feature.
- Without mentioning "CodeWhisperer", the description has very little context.

```suggestion
	"description": "CodeWhisper: pressing up or down does not move cursor when suggestion is active"
```

Reply:
Thanks, the changelog item has been updated.